.azure-pipelines/gpu-tests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     - bash: |
         python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
         pip install fairscale>=0.3.4
-        pip install "deepspeed>=0.4.3, !=0.4.4" # FIXME: bug with 0.4.4
+        pip install "deepspeed<0.4.4" # FIXME: bug with >= 0.4.4
```

Comment:
```suggestion
        pip install "deepspeed==0.4.3" # FIXME: bug with >= 0.4.4
```

Reply:
yep, that's fine too since it's only in our CI config file